stories/index.stories.js

```diff
@@ -1,19 +1,60 @@
-import React from 'react';
+import React, {Component} from 'react';
+import ReactDOM from 'react-dom';
 import {ButtonGroup} from 'react-bootstrap';
 import {storiesOf} from '@storybook/react';
 import {action} from '@storybook/addon-actions';
 import {linkTo} from '@storybook/addon-links';
 import {withInfo} from '@storybook/addon-info';
 
-import {Badge, Button, Breadcrumb} from '../components';
+import {Alert, Badge, Button, Breadcrumb} from '../components';
+import AlertDismissable from '../components/Alert';
 
 import '../less/styles.less';
 
 // Stories of components
+storiesOf('Alert', module)
+  .add('Danger Alert', withInfo('')(() => (
+    <div>
+      <AlertDismissable bsStyle="danger">
+        <h4> {'Delete this entity'} </h4>
```

Comment:
I wonder about this example. My understanding is that alerts are for information/feedback on an action after it's taken, not for prompts that initiate actions. So I'd think a modal would be more suited for this example. What do you think?

Reply:
Now that you put it that way, it does not make sense to have that example :) Updated the examples, couldn't think of a better one for Danger alert. What do you think will be more apt as an example here?!

Reply:
What you have now seems fine. :) It's really just an example to demonstrate behavior and we can always change it later.